### PR TITLE
添加米忽悠/崩坏3

### PR DESCRIPTION
**Pull request by DuckSoft**
图片
图片

### CODE REVIEW

File: data/mihoyo [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# 米忽悠
+domain:mihoyo.com
+
+# 崩坏3
+domain:bh3.com
```
Comment (iochen) [inline data/mihoyo:5]: since `www.bh3.com` also works, please modify it to
```
bh3.com
```

File: data/mihoyo [hunk 2]
```diff
@@ -0,0 +1,5 @@
+# 米忽悠
```
Comment (iochen) [inline data/mihoyo:1]: According to https://zh.wikipedia.org/wiki/%E7%B1%B3%E5%93%88%E6%B8%B8 ,
please modify it to 
```
#米哈游
```
or just remove this line

File: data/mihoyo [hunk 3]
```diff
@@ -0,0 +1,5 @@
...
+domain:mihoyo.com
```
Comment (iochen) [inline data/mihoyo:2]: since `www.mihoyo.com` also works, please modify it to
```
mihoyo.com
```

File: data/mihoyo (hunk 1 above)
Comment (iochen) [inline data/mihoyo:5]: Normally, `domain:` is omitted.

File: data/mihoyo (hunk 2 above)
Comment (iochen) [inline data/mihoyo:1]: According to https://zh.wikipedia.org/wiki/%E7%B1%B3%E5%93%88%E6%B8%B8 ,
it should be
```
#米哈游
```

File: data/mihoyo (hunk 1 above)
Comment (DuckSoft) [reply data/mihoyo:5]: I thought writing `domain:` will be more precise and more efficient when it comes to string matching.
Also I don't want to match something like `www.bh3.com.fake.example.com`.

File: data/mihoyo (hunk 2 above)
Comment (DuckSoft) [reply data/mihoyo]: Resolved.

File: data/mihoyo (hunk 1 above)
Comment (DuckSoft) [reply data/mihoyo]: Resolved.

**Comment by DuckSoft** [comment]
Ready to go.

File: data/mihoyo (hunk 1 above)
Comment (iochen) [reply data/mihoyo]: > I thought writing `domain:` will be more precise and more efficient when it comes to string matching.
> Also I don't want to match something like `www.bh3.com.fake.example.com`.

https://github.com/v2ray/domain-list-community/blob/2daf08f27670f3d985e72618ce9081b21ad0fbf3/main.go#L84-L88

There's no difference whether you add `domain:`.

**Pull request merged**